gouttelette/data/plugins/module_utils/utils.py

```diff
@@ -20,7 +20,6 @@
 
 from ansible.module_utils.common.dict_transformations import (
     camel_dict_to_snake_dict,
-    snake_dict_to_camel_dict,
```

Comment:
Same here. Just leave the imports as they are. We can cleanup the import path next.